docs/topics/intellij-idea-plugin-run-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-run-configuration_{context}"]
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project you import to IntelliJ IDEA.
+
+.Procedure
+
+. In *Projects* view, click the project you want to analyze.
+. On the left side of the screen, click the *{ProductName}* tab.
++
+If this is your first configuration, the run configuration panel is displayed on the right.
+
+. If this is not your first configuration, right-click on one of the configurations in the list and select *New configuration*.
++
+The run configuration panel is displayed on the right.
+
+. Complete the following configuration fields:
+
+** *mta-cli*: Click *Add*, select the filepath, and click *Save*.
+** *Input*: Click *Add* and do one of the following:
+
+*** Enter the input file or directory and press Enter.
+*** Click *Open File Explorer*, select the directory, and click *OK*.
+
+** *Target*: Select one or more target migration paths.
++
+[NOTE]
+====
+The location shown in the *Output* is set by the plugin.
+====
+
+. In the list of configurations, right-click on the new configuration and select *Run Analysis*.
++
+The *Console (MTA)* terminal emulator opens, displaying information about the progress of the analysis.
++
+When the analysis is completed, you can click either *Report* or *Results* below the name of the configuration file you ran.
```

Comment:
```suggestion
When the analysis is complete, you can click either *Report* or *Results* below the name of the configuration file you ran.
```
Also...you _can_? Meaning you don't have to? If you have to, or you should, then get rid of "you can".

Reply:
RH documentation uses the verb "can" consistently in this manner. Given that it is physically possible to do so but not obligatory, "can" and "may" are correct grammatically, but since only an authenticated authorized user _can_ reach this step. there's a case to be made for "can" (so, no change).

Reply:
+1 on _can_ instead of _may_. I was just wondering if it was obligatory.